templates/_navbar.html

```diff
@@ -50,3 +53,27 @@
 
 }
 </style>
+<script>
+$(function() {
+  let platform;
+
+  const url = document.URL.toLowerCase();
+  if (url.includes('qa.gigwork.net')) {
+    platform = 'Public QA';
```

Comment:
Can this be a configuration parameter?

Reply:
Fixed via https://github.com/bloomberg/pybossa-default-theme/pull/203/commits/0e58f67d7494852a7aea33a7e35fb68fd12bad57

Reply:
Wrapped platform identifier `div` with config option `DISPLAY_PLATFORM_IDENTIFIER` via https://github.com/bloomberg/pybossa-default-theme/pull/203/commits/5d3039c59976f91c516c8c1229fab7ce79214a1d